src/test/java/org/isf/patient/rest/PatientControllerTest.java

```diff
@@ -61,6 +61,9 @@
 	@Mock
     private PatientBrowserManager patientBrowserManagerMock;
 	
+	@Autowired
```

Comment:
As i mentioned before this  tests suite  pretends to be Spring independent, you would need to remove `@Autowired ` , instance the mapper manually, and provide the mapper in the setup method to the controller constructor like this:

```

    private static PatientMapper mapper = new PatientMapper();
	
    @Before
    public void setup() {
    	MockitoAnnotations.initMocks(this);
    	this.mockMvc = MockMvcBuilders
				.standaloneSetup(new PatientController(patientBrowserManagerMock, mapper))
   				.setControllerAdvice(new OHResponseEntityExceptionHandler())
   				.build();
    }
```